### PR TITLE
fix: toolbar shows all the options

### LINKED ISSUE #914: Impossible to hide / reorder options from toolbar
Good day,

It seems impossible to hide / re-order the options in the toolbar. I have tried to re-create example 10 (https://jpuri.github.io/react-draft-wysiwyg/#/demo), but the full set of options still show. 

  ```
<Editor
    wrapperClassName="demo-wrapper"
    editorClassName="demo-editor"
    toolbar={{
      inline: { inDropdown: true },
      list: { inDropdown: true },
      textAlign: { inDropdown: true },
      link: { inDropdown: true },
      history: { inDropdown: true }
    }}
  />
```

Schermafbeelding 2020-01-07 om 21 22 59

Maybe I miss something, but there seems to be no way to customise the options. I could hide options through CSS, but that does not solve the re-order problem.

Thank you.

### PR DESCRIPTION
**Pull request by nlopin**
Fixes #914

It looks like some naming collisions between two `toolbar` constants. If one of them is removed, everything works as expected.

I decided to get rid of one since it is redundant

### CODE REVIEW

File: src/Editor/index.js [hunk 1]
```diff
@@ -77,8 +77,7 @@ class WysiwygEditor extends Component {
     const newState = {};
     const { toolbar, editorState, contentState } = this.props;
     if (toolbar !== prevProps.toolbar) {
```
Comment (nlopin) [inline src/Editor/index.js]: @jpuri those objects are always different when user type inside the editor. Hence, we merge every render. Could you explain why we need to merge here?

**Comment by enBonnet** [comment]
Nice work @nlopin

File: src/Editor/index.js (hunk 1 above)
Comment (jpuri) [reply src/Editor/index.js]: Hey @nlopin : you are right I fixed this issue. Thanks 👍

**Comment by jpuri** [comment]
Thanks 👍

**Pull request merged**